### PR TITLE
docs: Add Translation to Assign Pods to Nodes using Node Affinity

### PR DESCRIPTION
**Pull request by irvifa**
Related to #22296

### CODE REVIEW

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    ```shell
+    NAME      STATUS    ROLES    AGE     VERSION        LABELS
+    worker0   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker0
+    worker1   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker1
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+1. Pilihkan salah satu dari Node yang ada dan tambahkan label pada Node tersebut.
+
+    ```shell
+    kubectl label nodes <your-node-name> disktype=ssd
```
Comment (sftim) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:40]:
```suggestion
    kubectl label nodes <nama-node-kamu> disktype=ssd
```

File: content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    worker2   Ready     <none>   1d      v1.13.0        ...,kubernetes.io/hostname=worker2
+    ```
+
+    Pada keluaran dari perintah di atas, kamu dapat melihat bahwa Node `worker0`
+    memiliki label `disktype=ssd`.
+
+## Menjadwalkan Pod menggunakan Afinitas Node
+
+Konfigurasi ini menunjukkan sebuah Pod yang memiliki afinitas node `requiredDuringSchedulingIgnoredDuringExecution`, `disktype: ssd`.
+Dengan kata lain, Pod hanya akan dijadwalkan hanya pada Node yang memiliki label `disktype=ssd`.
```
Comment (girikuncoro) [inline content/id/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:65]: nit: for aesthetics, "hanya" is repeated in the same sentence
```suggestion
Dengan kata lain, Pod hanya akan dijadwalkan pada Node yang memiliki label `disktype=ssd` saja.
```

**Pull request merged**